ros_package_template/README.md

```diff
@@ -18,38 +18,42 @@ Affiliation: Autonomous Systems Lab, ETH Zurich**
 ### Publications
 
 If you use this work in an academic context, please cite the following publication(s):
-	
+
 * P. Fankhauser, M. Bloesch, C. Gehring, M. Hutter, and R. Siegwart: **PAPER TITLE**. IEEE/RSJ International Conference on Intelligent Robots and Systems (IROS), 2015. ([PDF](http://dx.doi.org/10.3929/ethz-a-010173654))
 
+        @inproceedings{Fankhauser2015,
+            author = {Fankhauser, P\'{e}ter and Hutter, Marco},
+            booktitle = {IEEE/RSJ International Conference on Intelligent Robots and Systems (IROS)},
+            title = {{PAPER TITLE}},
+            publisher = {IEEE},
+            year = {2015}
+        }
+
 
 ## Installation
 
-### Dependencies
+### Installation from Packages
 
-- [Robot Operating System (ROS)](http://wiki.ros.org) (middleware for robotics),
-- [Eigen](http://eigen.tuxfamily.org) (linear algebra library),
-- [kindr](http://github.com/ethz-asl/kindr) (kinematics and dynamics library for robotics),
-- [Point Cloud Library (PCL)](http://pointclouds.org/) (point cloud processing),
-- [Grid Map](https://github.com/ethz-asl/grid_map) (grid map library for mobile robots),
-- [starleth_ros_common](http://bitbucket.org/ethz-asl-lr/c_starleth_ros_common) (common [ROS] packages for StarlETH robot).
-- [Gazebo](http://gazebosim.org/) (robot simulator)
+To install all packages from the this repository as Debian packages use
+
+    sudo apt-get install ros-indigo-...
+
+### Building from Source
 
-		sudo apt-get install ros-indigo-gazebo-ros-pkgs
+#### Dependencies
 
-	More information on the installation of [Gazebo] for [ROS] is given [here](http://gazebosim.org/tutorials?tut=ros_installing&cat=connect_ros),
-- [Schweizer-Messer](http://github.com/ethz-asl/Schweizer-Messer) (programming tools for robotics),
-	
-		sudo add-apt-repository ppa:ethz-asl/common
-		sudo apt-get update
-		sudo apt-get install schweizer-messer-common-dev schweizer-messer-timing-dev
+- [Robot Operating System (ROS)](http://wiki.ros.org) (middleware for robotics),
+- [Eigen] (linear algebra library)
+
+		sudo apt-get install libeigen3-dev
 
 
-### Building
+#### Building
 
-In order to install, clone the latest version from this repository into your catkin workspace and compile the package using
+To build from source, clone the latest version from this repository into your catkin workspace and compile the package using
 
 	cd catkin_workspace/src
-	git clone https://github.com/ethz-asl/PACKAGE_NAME.git
+	git clone https://github.com/ethz-asl/ros_package_template.git
 	cd ../
 	catkin_make
```

Comment:
We teach to use catkin build in the course, but the general case is still catkin_make.